gp/evolve.py

```
@@ -0,0 +1,40 @@
+import random
+from pathlib import Path
+from deap import base, creator, gp, tools, algorithms
+from .primitives import pset
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)